### PR TITLE
Update verification code for ldpVc in OID4VCI to support VCDM2

### PR DESCRIPTION
**Pull request by theosirian**

### CODE REVIEW

**Review by rschulman — APPROVED** [review]

File: src/oid4vci/error.rs [hunk 1]
```diff
@@ -11,31 +13,37 @@ pub enum Oid4vciError {
...
+    DecodeError(#[from] DecodeError),
+
     #[error("{_0}")]
     DidError(#[from] DidError),
 
     #[error("{_0}")]
     ContextMapError(#[from] FromContextMapError),
 
-    #[error("{0}")]
+    #[error("{_0}")]
```
Comment (cobward) [inline src/oid4vci/error.rs:46]: What does this change do?

File: src/oid4vci/mod.rs [hunk 2]
```diff
@@ -365,47 +366,33 @@ pub async fn oid4vci_exchange_credential(
                 JwtVcJson(response) => {
                     log::trace!("processing a JwtVcJson");
                     let rt = tokio::runtime::Runtime::new().unwrap();
```
Comment (Ryanmtate) [inline src/oid4vci/mod.rs:368]: You may be able to remove this runtime instance and specify the runtime in the export macro, i.e.:

```rust
#[uniffi::export(async_runtime = "tokio")]
pub async fn oid4vci_exchange_credential ...
```

Comment (Ryanmtate) [reply src/oid4vci/mod.rs:368]: This requires the `tokio` feature in uniffi, which was added when #43 was merged.

```toml
uniffi = { version = "0.28.1", features = ["cli", "tokio"] }
```

**Review by Ryanmtate — APPROVED** [review]

File: src/oid4vci/error.rs (hunk 1 above)
Comment (theosirian) [reply src/oid4vci/error.rs:46]: Not sure what the difference between `{0}` and `{_0}` is.
I changed them all to be consistent inside the enum.

**Pull request merged**